shared/profile/wot-author/index.tsx

```diff
@@ -30,6 +30,23 @@ export type Question1Answer = {
   verificationType: WebOfTrustVerificationType
 }
 
+type ReviewAction = 'accept' | 'propose' | 'reject'
+
+export type ReviewProps = {
+  error?: string
+  firstDraft: boolean
+  onAccept: () => void
+  onProposeEdits: () => void
+  onReject: () => void
+  otherText: string
+  proofs: Array<Proof>
+  statement: string
+  verificationType: WebOfTrustVerificationType
+  voucheeUsername: string
+  voucherUsername: string
+  waiting?: undefined | ReviewAction
```

Comment:
`?` means `undefined`

```suggestion
  waiting?: ReviewAction
```